### ISSUE TITLE
Not correct resolution for iPhone 6 and iPhone 6 Plus.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#504]
```
---------------------------------------------------------------------------
NOTE: This issue tracking system is for Google Analytics developer products
only.
If you are not a developer/programmer visit:
http://www.google.com/analytics/support.html
---------------------------------------------------------------------------
Name of affected component: Mobile Tracking

Name of related library and version, if applicable (e.g. Android, iOS,
Snippets, etc.): iOs 3.08 (latest sdk)

Issue summary:
sdk, doesn't detect screen resolutions for iPhone 6 and iPhone 6Plus.

Steps to reproduce issue:
1. Launch any application on iPhone 6 or iPhone 6 Plus with GA inside.
2. Wait for statistics.
3. See statistics for screen resolutions in GA.

Expected output:
for iPhone 6 - 375 × 667,
for iPhone 6 Plus - 414 × 736.

Actual results:
320 х 568

Notes:
I think, that the problem is in using UIScreen.mainScreen.bounds and not 
UIScreen.mainScreen.nativeBounds for ios8 and later.
```

Original issue reported on code.google.com by `rtishc...@peregrins.com` on 3 Oct 2014 at 9:51

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
Original comment by `pfrise...@google.com` on 10 Oct 2014 at 6:25
- Added labels: **Library-iOS**

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem with my existing app. but when i tried this new app 
which is created using Xcode6, that method work fine.

So I think, this issue is for existing app which are compatible for iOS8.
I though this is some project configuration issue when we converted project 
compatible with Xcode6.




```

Original comment by `deepaksa...@gmail.com` on 7 Nov 2014 at 5:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `pfrise...@google.com` on 14 Nov 2014 at 9:46
- Changed state: **Acknowledged**

**Comment by GoogleCodeExporter** [comment]
```
Addressed in v3.11
See: https://developers.google.com/analytics/devguides/collection/ios/changelog
```

Original comment by `pfrise...@google.com` on 30 Apr 2015 at 6:26
- Changed state: **Fixed**
